src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -181,6 +181,7 @@ class FlowHelper {
      */
     static FlowEndpointV2 toFlowEndpointV2(FlowEndpointPayload endpoint) {
         new FlowEndpointV2(endpoint.datapath, endpoint.portNumber, endpoint.vlanId,
+                (endpoint.innerVlanId ? endpoint.innerVlanId : 0),
```

Comment:
Can be simplified to `endpoint.innerVlanId ?: 0`

Reply:
refactord